quesma/comment_metadata/comment_metadata.go

```diff
@@ -0,0 +1,76 @@
+// Copyright Quesma, licensed under the Elastic License 2.0.
+// SPDX-License-Identifier: Elastic-2.0
+package comment_metadata
+
+import (
+	"fmt"
+	"net/url"
+	"regexp"
+)
+
+const ElasticFieldName = "fieldName"
+
+const commentMetadataVersion = "1"
+const metadataPrefix = "quesmaMetadata:"
+const versionParameter = "v"
```

Comment:
I think  we could ditch this additional const `versionParameter` and just use `metadataVersion` and let it be `v1`  for simplicity. parsing will become more clear